pkg/virt-handler/seccomp/seccomp.go

```diff
@@ -0,0 +1,78 @@
+package seccomp
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/containers/common/pkg/seccomp"
+)
+
+// Install seccomp, kubeletRoot should be passed in format: /proc/1/root/var/lib/kubelet/
+func InstallPolicy(kubeletRoot string) error {
+	const errMsgFormat string = "failed to install default seccomp profile: %v"
+
+	installPath := filepath.Join(kubeletRoot, "seccomp/kubevirt")
```

Comment:
Looks like `safepath` would be pretty well suited for this function.
It is not needed for now, as all parts of the path seem to be hardcoded in consts, but if any part of it ever becomes configurable we'll definitely have to switch to using `safepath`.

Reply:
I myself was thinking about it. I will follow up in the next PR.